packages/cozy-pouch-link/src/CozyPouchLink.js

```diff
@@ -539,7 +539,12 @@ class PouchLink extends CozyLink {
 
   async deleteDocument(mutation) {
     const res = await this.dbMethod('remove', mutation)
-    return parseMutationResult(mutation.document, res)
+    const document = {
+      ...mutation.document,
+      _rev: res.rev,
+      _deleted: true
+    }
```

Comment:
To be equal to DocumentCollection.destroy(), you should also add the `_id` https://github.com/cozy/cozy-client/blob/master/packages/cozy-stack-client/src/DocumentCollection.js#L379